src/libpmemfile-posix/data.c

```diff
@@ -101,6 +101,15 @@ block_cache_insert_block(struct ctree *c, struct pmemfile_block_desc *block)
 	return 0;
 }
 
+static void
+block_cache_insert_block_noret(struct ctree *c,
+		struct pmemfile_block_desc *block)
+{
+	int err = block_cache_insert_block(c, block);
+	if (err)
+		/* not possible */
```

Comment:
It was not obvious at first why it is "not possible".
Perhaps this function could be renamed to `block_cache_insert_block_in_tx`,
and the body could be something like:
```
if (pmemobj_tx_stage() != TX_STAGE_WORK)
   FATAL("fatal!!!!!");
(void) block_cache_insert_block(c, block);
```